react/src/components/ImageEnvironmentSelectFormItems.tsx

```diff
@@ -163,19 +188,27 @@ const ImageEnvironmentSelectFormItems: React.FC<
           getImageFullName(image),
       )
     ) {
-      const nextNewImage = nextEnvironmentGroup?.images[0];
-      if (nextNewImage) {
-        form.setFieldsValue({
-          environments: {
-            environment: nextEnvironmentName,
-            version: getImageFullName(nextNewImage),
-            image: nextNewImage,
-          },
-        });
-      }
+      nextNewImage = nextEnvironmentGroup?.images[0];
+    } else {
+      // if image info is partially set, fill out image info
+      nextNewImage = _.find(
+        nextEnvironmentGroup?.images,
+        (image) =>
+          form.getFieldValue('environments')?.version ===
+          getImageFullName(image),
+      );
```

Comment:
Could you let me know how to reproduce the bug you solved by adding these lines??